### DISCUSSION TITLE
Does this work with dynamic images?

### DISCUSSION THREAD

**Discussion by deodat** [ivodolenc/nuxt-lazysizes#13] [Q&A] [answered]
Hello,

I wonder if this will work with "dynamic" images, like ones which are loaded from a CMS for example?
All the examples are static ones, with images loaded from `assets` folder.

Thanks,

Regards,

Deodat

**Comment by ivodolenc** [comment]
Have you tried some example already? I believe it should work without too much hassle. The dynamic part is up to you, lazysizes loads images when they enter in the viewport.

Try to explain in more detail what you want to achive.

**Comment by deodat** [comment, marked as answer]
Hello @ivodolenc,

Indeed, I tried the  blurUp example with my Storyblok images like this:
```
      <div class="mediabox">
        <img
          :data-src="post.content.featured_image.filename"
          :data-lowsrc="
            $options.filters.transformImage(
              post.content.featured_image.filename,
              '20x10'
            )
          "
          data-sizes="auto"
          class="mediabox-img lazy"
        />
      </div>
```
and it works like a charm :)

**Reply by ivodolenc** [reply]
Awesome! I'm glad it worked!
